lib/frontman/commands/init.rb

```diff
@@ -15,7 +16,7 @@ def init(path)
 
       target_dir = File.join(Dir.pwd, path == '.' ? '' : path)
 
-      unless allowed_to_modify_dir?(target_dir)
+      unless !options[:force].nil? || allowed_to_modify_dir?(target_dir)
```

Comment:
```suggestion
      unless options[:force] || allowed_to_modify_dir?(target_dir)
```

I think this could also work like this, what do you think?

Reply:
Yes, this syntax would work if we set the `:boolean` type for this option, to make sure `Thor` uses it as a flag. I left a suggestion to address that 👍